src/points/saga.test.ts

```diff
@@ -108,13 +117,13 @@ describe('getHistory', () => {
       .put(
         getHistorySucceeded({
           appendHistory: false,
-          newPointsHistory: MOCK_HISTORY_RESPONSE.data,
+          newPointsHistory: MOCK_HISTORY_RESPONSE.data.slice(0, 2),
```

Comment:
Why are we doing this now?

Reply:
Since we now filter out history records whose activityIds are unknown/invalid.

Reply:
Ok! Should we maybe introduce a `const MOCK_SUPPORTED_HISTORY = MOCK_HISTORY_RESPONSE.filter(...)` to make it more obvious? possibly with a little comment.

Reply:
Yeah! good idea.